packages/web/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@lmc-eu/spirit-web",
+  "version": "0.1.0",
+  "description": "CSS and JS for web apps built with Spirit Design System",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
```

Comment:
I think we should prepare package during build for better usage like it was in colors - https://github.com/lmc-eu/spirit-design-system/blob/main/packages/colors/package.json, eg. using `files` define which files or directories should be in final package. For usage like `@lmc-eu/spirit-web/scss` or `@lmc-eu/spirit-web/css`

Reply:
This one is a little bit tough. Currently it seems that `dist` remains in final packages when I test publishing with `npm pack`. Does Lerna take control over it? Or do we need to use `files` and `.npmignore` instead of `dist`?

Paths are tackled in #18 but now I'm not really sure if `dist` will remain in final packages or not. Docs in #18 suppose they will not.

Reply:
As discussed today, we will try to go with `dist` and see what happens.